src/Jackett.Common/Indexers/Libble.cs

```diff
@@ -0,0 +1,321 @@
+                            {"password", configData.Password.Value},
+                            {"login", "Login"}
+                        };
+
+            var result = await RequestLoginAndFollowRedirect(LoginUrl, pairs, null, true, SearchUrl, LandingUrl, true);
+            await ConfigureIfOK(result.Cookies, result.Content?.Contains("logout.php") == true,
+                () =>
+                {
+                    var parser = new HtmlParser();
+                    var dom = parser.ParseDocument(result.Content);
+                    var warningNode = dom.QuerySelector("#loginform > .warning");
+                    var errorMessage = warningNode?.TextContent.Trim().Replace("\n\t", " ");
+                    throw new ExceptionWithConfigData(errorMessage, configData);
+                });
+
+
+            SaveConfig();
+
+            return IndexerConfigurationStatus.RequiresTesting;
+        }
+
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            // https://libble.me/torrents.php?searchstr=the+used&taglist=&tags_type=1&order_by=time&order_way=desc
+            var releases = new List<ReleaseInfo>();
+            var searchString = query.GetQueryString();
+            var prevCook = CookieHeader + "";
+            var searchUrl = SearchUrl;
+
+            var searchParams = new Dictionary<string, string> { };
+            var queryCollection = new NameValueCollection { };
+
+            // Skip search params if its a test, ask for all torrents
+            if (!query.IsTest)
+            {
+                if (!string.IsNullOrWhiteSpace(query.ImdbID))
+                {
+                    queryCollection.Add("cataloguenumber", query.ImdbID);
+                }
+                else if (!string.IsNullOrWhiteSpace(searchString))
+                {
+                    queryCollection.Add("searchstr", searchString);
+                }
+            }
+
+            // Filter Categories
+            if (query.HasSpecifiedCategories)
+            {
+                foreach (var cat in MapTorznabCapsToTrackers(query))
+                {
+                    queryCollection.Add("filter_cat[" + cat.ToString() + "]", "1");
+                }
+            }
+
+            if (query.Artist != null)
+            {
+                queryCollection.Add("artistname", query.Artist);
+            }
+
+            if (query.Label != null)
+            {
+                queryCollection.Add("recordlabel", query.Label);
+            }
+
+            if (query.Year != null)
+            {
+                queryCollection.Add("year", query.Year.ToString());
+            }
+
+            if (query.Album != null)
+            {
+                queryCollection.Add("groupname", query.Album);
+            }
+
+            searchUrl += "?" + queryCollection.GetQueryString();
+
+            var searchPage = await PostDataWithCookiesAndRetry(searchUrl, searchParams, CookieHeader);
```

Comment:
If you are not doing "anything special" with the cookies you don't need to pass this parameter.